frontend/src/Components/SignupForm.jsx

```
@@ -0,0 +1,85 @@
+          <div className="form-group">
+            <label>Email</label>
+            <input
+              type="email"
+              placeholder="Enter your email"
+              value={formData.email}
+              onChange={(e) => setFormData({...formData, email: e.target.value})}
+            />
+          </div>
+
+          <div className="form-group">
+            <label>Password</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)